### DISCUSSION TITLE
FAQ: To what extent should we include rich content?

### DISCUSSION THREAD

**Discussion by Rabadash8820** [olivierlacan/keep-a-changelog#519] [Q&A]
Is it okay to include "rich" content like images, links to other projects, blockquotes, etc. in the changelog? While these things might make it more "enjoyable" to read, I can also see where they might make it harder for users to copy/paste and more complex for machines to parse. Whatever is considered the best practice with regards to rich content, I feel like that should be included in the FAQ.

**Comment by oxr463** [comment]
Perhaps such content should go on a project blog, separate from the need-to-know release information. You could then link to the blog post from the Changelog.

**Comment by Rabadash8820** [comment]
@oxr463 Thanks for the reply. That makes sense to me. I've definitely come to feel like the changelog should be kept as text-based as possible; one should be able to view it in Notepad without any loss of information or context. I realize this Issue is pretty opinion-based, so idk if @olivierlacan or another maintainer wants to wait for any other input before closing it. I still feel like this is worth mentioning in the project FAQs, but at least now this issue is here to be searched by future googlers.
